dm/dm/master/openapi.go

```diff
@@ -901,6 +901,13 @@ func (s *Server) DMAPUpdateTaskTemplate(c *gin.Context, taskName string) {
 	c.IndentedJSON(http.StatusOK, task)
 }
 
+// DMAPIGetClusterID return cluster id of dm cluster
+func (s *Server) DMAPIGetClusterID(c *gin.Context) {
```

Comment:
may add a more common api like `/api/v1/cluster/info` which contains id (only contains id now) so we can add more info to it later.

Reply:
> may add a more common api like `/api/v1/cluster/info` which contains id (only contains id now) so we can add more info to it later.

As now we have `/cluster/version`, so i add `/cluster/id`.We can add `/cluster/info` when we need and remove those two in the future.

Reply:
> remove those two in the future.

as this is an OpenAPI Spec and open source , i think maybe we should do less break changes 😂

Reply:
I will change it to `cluster/info`.